### PR TITLE
⏱️ Reduce e2e time

### PR DESCRIPTION
**Pull request by MontaGhanmy**

### CODE REVIEW

File: tdrive/backend/node/test/e2e/documents/documents-browser.spec.ts [hunk 1]
```diff
@@ -25,75 +28,63 @@ describe("The Documents Browser Window and API", () => {
...
   });
 
   describe("My Drive", () => {
-
-    it("Should successfully upload filed to the 'Shared Drive' and browse them", async () => {
-      const myDriveId = "user_" + currentUser.user.id;
-      const result = await currentUser.uploadAllFilesOneByOne(myDriveId);
-
+    it("Should successfully upload filed to the 'My Drive' and browse them", async () => {
       expect(result).toBeDefined();
```
Comment (shepilov) [inline tdrive/backend/node/test/e2e/documents/documents-browser.spec.ts]: move then this expects also to beforeEach, as far as they are not connected to this test anymore

File: tdrive/backend/node/test/e2e/documents/documents-browser.spec.ts [hunk 2]
```diff
@@ -25,75 +28,63 @@ describe("The Documents Browser Window and API", () => {
       ],
     });
     currentUser = await UserApi.getInstance(platform);
+    sharedWIthMeFolder = "shared_with_me";
+    myDriveId = "user_" + currentUser.user.id;
+    result = await currentUser.uploadAllFilesOneByOne(myDriveId);
```
Comment (shepilov) [inline tdrive/backend/node/test/e2e/documents/documents-browser.spec.ts]: rename it to files?

**Comment by shepilov** [comment]
@MontaGhanmy why it's still in the "draft" mode? do you want to merge it?

**Comment by MontaGhanmy** [comment]
> @MontaGhanmy why it's still in the "draft" mode? do you want to merge it?

@shepilov please approve it now, I'll merge it.

**Review by shepilov — APPROVED** [review]

**Pull request merged**